### PR TITLE
Feature/pelagos 4543 add home page for hri

### PR DESCRIPTION
**Pull request by praneethpr**

### CODE REVIEW

File: src/Controller/DefaultController.php [hunk 1]
```diff
@@ -114,8 +71,39 @@ private function getProjectDirectorList(array $leadershipPersons): array
...
+
+            foreach ($fundingCycles as $fundingCycle) {
+                $fundingCycleList[] = array(
+                    'id' => $fundingCycle->getId(),
+                    'name' => $fundingCycle->getName(),
+                    'researchGroups' => $this->getResearchGroupsArray($fundingCycle)
+                );
+            }
+
+            return $this->render("Default/$customTemplate", array(
```
Comment (mickel1138) [inline src/Controller/DefaultController.php:101]: I figured out why there was an error, if this template doesn't exist, it breaks and doesn't even go to the default.
We should again rethink the switching mechanism, but switching by filter is not the same as switching by name/site/template.

Comment (praneethpr) [reply src/Controller/DefaultController.php:101]: We can talk about this

**Review by fightingtexasaggie — COMMENTED** [review]
Are we going to use a separate index on this subsite the same way we do for GRP? If not, we'd show all datasets on the map-search page.

**Comment by praneethpr** [comment]
> Are we going to use a separate index on this subsite the same way we do for GRP? If not, we'd show all datasets on the map-search page.

Need a separate ticket for creating HRI search index

**Review by fightingtexasaggie — APPROVED** [review]
I'm good with this initial site, and have tested the components (dif/submission/review/dataland/stats/monitoring). We still have to decide on the index, but as Praneeth said, that should be a separate ticket.

**Pull request merged**